galsim/phase_psf.py

```diff
@@ -1432,6 +1447,9 @@ def __repr__(self):
         s += ", aper=%r"%self._psf.aper
         if any(screen.aberrations):
             s += ", aberrations=[" + ",".join(str(ab) for ab in screen.aberrations) + "]"
+        if screen.annular_zernike:
+            s += ", annular_zernike=True"
+            s += ", obscuration=%r"%self.obscuration
```

Comment:
That was on purpose.  No need to print out the obscuration unless the Zernikes are annular.